hooks/006-add-gdm.chroot

```diff
@@ -82,6 +82,7 @@ rm -f /etc/systemd/user/default.target.wants/pipewire*
 rm -f /usr/lib/systemd/user/org.freedesktop.IBus.session.GNOME.service
 rm -f /usr/lib/systemd/user/org.freedesktop.IBus.session.generic.service
 rm -f /usr/lib/systemd/user/gnome-session.target.wants/org.freedesktop.IBus.session.GNOME.service
+rm -f /usr/lib/systemd/user/colord-session.service
```

Comment:
I suspect `/usr/share/dbus-1/services/org.freedesktop.ColorHelper.service` is the main file that needs to be removed if we want to move this service into the session snap's sandbox. If that's still present, dbus-daemon will likely still execute it unconfined.